application/common/components/Emailer.php

```diff
@@ -493,30 +493,32 @@ public function shouldSendMfaEnabledMessageTo($user, $mfaEventType)
     }
 
     /**
-     * Whether the user just deleted an MFA option and there is still one or more left
+     * Whether the user just deleted a verified MFA option and there is still one or more left
      *
      * @param User $user (assumes the user instance has already been refreshed)
      * @param string Mfa::EVENT_TYPE_*
      * @return bool
```

Comment:
Looks like you'll also want to add an `@param` on each of these functions for the new `$mfa` parameter.